policies/templates/gcp_external_ip_access.yaml

```diff
@@ -23,8 +23,7 @@ spec:
   crd:
     spec:
       names:
-        kind: GCPExternalIpAccessConstraint
-        listKind: GCPExternalIpAccessConstraintList
+        kind: GCPExternalIpAccessConstraintV1
         plural: GCPExternalIpAccessConstraints
```

Comment:
Doesn't the plural also need to be versioned?

Reply:
Done.

Also switched it to lowercase and removed singular field. 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#customresourcedefinitionnames-v1beta1-apiextensions-k8s-io mentioned that plural must be all lowercase and that singular defaults to lowercased <kind>